src/motion-queue.h

```diff
@@ -105,9 +105,10 @@ class MotionQueue {
   // Shutdown. If !flush_queue: immediate, even if motors are still moving.
   virtual void Shutdown(bool flush_queue) = 0;
 
-  // Fill the argument with the current absolute position in loops
-  // for each motor.
-  virtual void GetMotorsLoops(MotorsRegister *absolute_pos_loops) = 0;
+  // Return the number of motion segments enqueued after
+  // the one currently executing and its already executed loops.
+  // buffer_size will never be 0 except if nothing has been yet enqueued.
```

Comment:
This comment is now not accurate anymore (there is no buffer_size).

Also, it is not quite clear from the description: While something is executing, what does this method return ? I would expect '1' (as there is still one element being worked on) and the head item progress is set to some value. While Zero would be returned when the queue is fully executed and empty (and in that case, we don't expect head_item_progress to be updated for instance).